Description: Contextfree fails to cross build from source, because it strips with the
 wrong strip. Such stripping also breaks DEB_BUILD_OPTIONS=nostrip as
 well as generation of -dbgsym packages. stripping is best deferred to
 dh_strip. 
Author: Helmut Grohne <helmut@subdivi.de> Debian QA
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -134,7 +134,6 @@
 
 cfdg: $(OBJS)
 	$(LINK.o) $^ $(LINKFLAGS) -o $@
-	strip $@
 
 
 #
